.github/scripts/shared/labelable.ts

```diff
@@ -65,6 +65,26 @@ export async function addLabelToLabelable(
   });
 }
 
+// This function adds label by id to a labelable object (i.e. a pull request or an issue)
+export async function addLabelByIdToLabelable(
```

Comment:
Maybe we can use this new `addLabelByIdToLabelable` function inside `addLabelToLabelable`? (to avoid code duplication)

Reply:
I agree! Thank you for catching this.